npm/ng-packs/packages/feature-management/src/lib/components/feature-management/feature-management.component.ts

```diff
@@ -125,7 +139,7 @@ export class FeatureManagementComponent
       .subscribe(() => {
         this.visible = false;
 
-        this.toasterService.success('AbpFeatureManagement::Saved');
+        this.toasterService.success('AbpFeatureManagement::SavedSuccessfully');
         if (!this.providerKey) {
```

Comment:
I see that we have SavedSuccessfully key only for AbpSettingManagement and AbpUi scopes. So, I am updating this to AbpUi.